src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,17 +1,105 @@
-import React from 'react';
+import React, { useState } from 'react';
+import cn from 'classnames';
+import { getMovie } from '../../api';
 import './FindMovie.scss';
+import { Movie } from '../../types/Movie';
+import { MovieData } from '../../types/MovieData';
+import { MovieCard } from '../MovieCard';
+
+const INITIAL_STATE: Movie = {
+  title: '',
+  description: '',
+  imdbUrl: '',
+  imgUrl: '',
+  imdbId: '',
+};
+
+type Props = {
+  addMovie: (movie: Movie) => void,
+  isInList: (imdbId: string) => boolean,
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovie, isInList }) => {
+  const [movie, setMovie] = useState<Movie>(INITIAL_STATE);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isFound, setIsFound] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [isTyping, setIsTyping] = useState(false);
+  const [query, setQuery] = useState('');
+
+  const normalizeData = (data: MovieData): Movie => {
+    const {
+      Title, Plot, Poster, imdbID,
+    } = data;
```

Comment:
Variables are called with a lowercase letter
```suggestion
    const {
      title, 
      plot, 
      poster, 
      imdbID,
    } = data;
```